docs/source/structured_config.rst

```diff
@@ -332,6 +332,21 @@ To work around it, use SI and II described below.
     >>> assert conf.b == 100
     >>> assert conf.c == 100
 
+Runtime type validation is not performed on interpolated variables, see below.
```

Comment:
I think this is a bit better (type validation IS performed on assignment).
---
Runtime type validation is not performed on values returned by interpolation, e.g:

```python
@dataclass
class Interpolation:
    int_key: int = II("str_key")
    str_key: str = "string"

cfg = OmegaConf.structured(Interpolation)
assert cfg.int_key == "string"
```

Reply:
Sure, updated :)